spec/metanorma/cli/generator_spec.rb

```diff
@@ -60,6 +60,45 @@
         expect(output).to include("Sorry, could not generate the document!")
       end
     end
+
+    context "with local template" do
+      it "success for existing template" do
+        document = "./tmp/my-local-document"
+        template = "./templates"
+
+        capture_stdout {
+          Metanorma::Cli::Generator.run(
+              document,
+              type: "csd",
+              overwrite: true,
+              doctype: "standard",
+              template: template,
+              )
+        }
+
+        expect_document_to_include_base_templates(document)
+        expect(file_exits?(document, "README.adoc")).to be_truthy
+        expect(file_exits?(document, "cc-document.adoc")).to be_truthy
```

Comment:
In mn-templates-csd the main file per template has bee changed to `document.adoc` (removed `cc-` prefix).

Reply:
Done